lib/avo/resources/base.rb

```diff
@@ -359,7 +361,7 @@ def divider(label = nil)
       end
 
       def hydrate(...)
-        super(...)
+        super
```

Comment:
Will `super` have access to all the method arguments without explicitly passing them?

Reply:
Yes, super has access to all the method args without explicitly passing them. This change was done by standardrb when I linted btw.
![image](https://github.com/user-attachments/assets/96badb08-f40a-41b7-a165-8ee20d3582b9)